sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs

```diff
@@ -350,7 +350,7 @@ public virtual Response JobStatus(Guid jobId, bool? showStats = null, RequestCon
         /// </code>
         /// 
         /// </remarks>
-        public virtual async Task<Operation<BinaryData>> SubmitJobAsync(WaitUntil waitUntil, RequestContent content, RequestContext context = null)
+        public virtual async Task<Operation> SubmitJobAsync(WaitUntil waitUntil, RequestContent content, RequestContext context = null)
```

Comment:
@heaths, could you confirm that there is no return-type from the `SubmitJob` LRO?

Reply:
yes, I confirmed that SubmitJob will not return an value.

Reply:
That is correct: it does not return data.